.github/workflows/codespell.yml

```diff
@@ -16,6 +16,6 @@ jobs:
       - name: codespell
         uses: codespell-project/actions-codespell@master
         with:
-          skip: .git
+          skip: .git,vendor
```

Comment:
Can please add go.mod and go.sum too 

refer https://github.com/csi-addons/kubernetes-csi-addons/runs/4604315553?check_suite_focus=true

Reply:
ok, done!